kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/DescrNameViewModel.java

```diff
@@ -62,20 +64,59 @@ public class DescrNameViewModel extends FormViewModel {
 
     public static final String DESCRIPTION_NAME_TYPE = "descrNameType";
 
+    public static final String IS_INVALID = "isInvalid";
 
     public DescrNameViewModel() {
         super(); // defaults to View mode
-        addProperty(NAME_TEXT, "")
+                addProperty(NAME_TEXT, "")
+                .addValidator(NAME_TEXT, "Name Text", (ReadOnlyStringProperty prop, ViewModel vm) -> {
+                    if (prop.isEmpty().get()) {
+                        return new ValidationMessage(NAME_TEXT, MessageType.ERROR, "${%s} is required".formatted(NAME_TEXT));
+                    }
+                    return VALID;
+                })
```

Comment:
Can we use Cognitive's new feature to simplify verbose Validators and allows you to add more validation messages? (Note: the new feature was created in Cognitive 1.2.0)

Can you bump up there version of Cognitive 1.3.0?

The above code can be rewritten as the following:
```java
.addValidator(NAME_TEXT, "First Name", (ReadOnlyStringProperty prop, ValidationResult validationResult, ViewModel viewModel) -> {
    if (prop.isEmpty().get()) {
        validationResult.error("${%s} is required".formatted(NAME_TEXT));
    }
});
```

While you only have one validation it is still less verbose. The feature allows you to create multiple validation messages.

Reply:
updated code to use 1.2.0 feature.
Updated pom.xml to use cognitive 1.3.0 version.